os/arch/arm/src/sidk_s5jt200/src/s5jt200_tash.c

```diff
@@ -284,7 +284,7 @@ int board_app_initialize(void)
 	}
 #endif /* CONFIG_SIDK_S5JT200_AUTOMOUNT_ROMFS */
 
-#ifdef CONFIG_SIDK_S5JT200_AUTOMOUNT_USERFS
+#if defined(CONFIG_FS_SMARTFS) && defined(CONFIG_SIDK_S5JT200_AUTOMOUNT_USERFS)
```

Comment:
Let's move this additional conditional to Kconfig like artik053. This functionality is enough with CONFIG_SIDK_S5JT200_AUTOMOUNT_USERFS. We should make a dependancy it with FS_SMARTFS at Kconfig.

Reply:
Ok.